client/home/lib/billing/components/subscriptionsection.coffee

```diff
@@ -19,6 +20,7 @@ module.exports = class SubscriptionSection extends React.Component
 
     <DashboardSection title='Koding Subscription'>
       <Subscription {...subscriptionProps} />
+      <AddonsBanner />
```

Comment:
I believe showing AddonsBanner should be tied to any condition to show it to user.
We might provide you more info about when it should be showed or not